assets/php/class-wgpb-block-library.php

```diff
@@ -283,6 +285,7 @@ public static function print_script_settings() {
 			'placeholderImgSrc' => wc_placeholder_img_src(),
 			'min_height'        => wc_get_theme_support( 'featured_block::min_height', 500 ),
 			'default_height'    => wc_get_theme_support( 'featured_block::default_height', 500 ),
+			'catalogSize'       => $product_counts->publish,
```

Comment:
I wonder if it might be better to let the php code decide if it's a "large" catalog or not, and just return a boolean here, rather than have the js code decide.